lib/vmdb/config/activator.rb

```diff
@@ -1,17 +1,10 @@
 module VMDB
-  class Config
+  module Config
```

Comment:
Why change this from a class to a module? Couldn't we also remove the `Config` namespace from these, maybe put them under `/lib/vmdb/settings/`?

As it stands you'll need to change https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/settings.rb#L51